appengine/flexible/tasks/README.md

```diff
@@ -0,0 +1,105 @@
+# Google Cloud Tasks App Engine Queue Samples
+
+Sample command-line program for interacting with the Cloud Tasks API
+using App Engine queues.
+
+App Engine queues push tasks to an App Engine HTTP target. This directory
+contains both the App Engine app to deploy, as well as the snippets to run
+locally to push tasks to it, which could also be called on App Engine.
+
+`app_engine_queue_snippets.py` is a simple command-line program to create tasks
+to be pushed to the App Engine app.
+
+`main.py` is the main App Engine app. This app serves as an endpoint to receive
+App Engine task attempts.
+
+`app.yaml` configures the App Engine app.
+
+
+## Prerequisites to run locally:
+
+Please refer to [Setting Up a Python Development Environment](https://cloud.google.com/python/setup).
+
+## Authentication
```

Comment:
Can you just link to our central auth docs?

Reply:
You mean just link to this page: https://cloud.google.com/docs/authentication/ ? It seems way, way too complex for users who just want to run the samples. I'm not even clear on where I would click on this page in order to figure out how to run them.

Reply:
Specifically I meant here: https://cloud.google.com/docs/authentication/getting-started

We should no longer recommend gcloud auth to users.

Reply:
Okay, I'll replace these sections with that link.